ast_convenience.ml

```diff
@@ -53,7 +53,7 @@ let tuple ?loc ?attrs = function
   | xs -> Exp.tuple ?loc ?attrs xs
 let cons ?loc ?attrs hd tl = constr ?loc ?attrs "::" [hd; tl]
 let list ?loc ?attrs l = List.fold_right (cons ?loc ?attrs) l (nil ?loc ?attrs ())
-let str ?loc ?attrs s = Exp.constant ?loc ?attrs (Pconst_string (s, None))
+let str ?loc ?attrs s = Exp.constant ?loc ?attrs (Pconst_string (s, Location.none, None))
```

Comment:
Shouldn't the inner location of the Pconst_string be `loc` or `!default_loc`?
This location corresponds to the string content in expression like:
```ocaml
{alongprefix|content|alongprefix}
(*           ^^^^^^^
 this is the Pconst_string location
*)
```